### DISCUSSION TITLE
Use "CreativeGodmode" instead

### DISCUSSION THREAD

**Discussion by GUZZ07** [Pryaxis/TShock#2076] [Ideas]
I noticed that there's a "creativeGodmode" field of Terraria.Player, and analyzed it.
The following code can provide a real godmode instead the existed fake one

`var power = CreativePowerManager.Instance.GetPower<CreativePowers.GodmodePower>();`
`power.SetEnabledState(player.Index, true);`

**Comment by hakusaro** [comment]
I'm pretty sure it's impossible for us to use this outside of SSC.
GitHub <notifications@github.com> wrote:
“I noticed that there's a "creativeGodmode" field of Terraria.Player, and analyzed it.
 The following code can provide a real godmode instead the existed fake one

var power = CreativePowerManager.Instance.GetPower<CreativePowers.GodmodePower>();
 power.SetEnabledState(player.Index, true);”
